loky/backend/context.py

```diff
@@ -126,44 +83,37 @@ def cpu_count(only_physical_cores=False):
     any other way such as: process affinity, restricting CFS scheduler policy
     or the LOKY_MAX_CPU_COUNT environment variable. If the number of physical
     cores is not found, return the number of logical cores.
- 
+
     It is also always larger or equal to 1.
     """
-    # TODO: use os.cpu_count when dropping python 2 support
-    try:
-        cpu_count_mp = mp.cpu_count()
-    except NotImplementedError:
-        cpu_count_mp = 1
+    cpu_count_mp = os.cpu_count() or 1
```

Comment:
From [docs](https://docs.python.org/3/library/os.html#os.cpu_count) for `os.cpu_count()` it can return `None` if undetermined, and without `or 1` `mypy/pylance` complains about subsequent usages of `cpu_count_mp` as having type Optional[int, None].
So yes, `or 1` is necessary.